app/templates/edit_admin_user.html

```diff
@@ -0,0 +1,97 @@
+{% extends "_base_page.html" %}
+{% block page_title %}
+  Edit details of admin user: {{ admin_user.emailAddress }} – Digital Marketplace admin
+{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": url_for('.index'),
+        "label": "Admin home"
+      },
+      {
+        "label": "Edit an admin user"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+  {% with messages = get_flashed_messages(with_categories=true) %}
+  {% if messages %}
+      {% for category, message in messages %}
+          {%
+          with
+            message = message,
+            type = "destructive" if category == 'error' else 'success'
+          %}
+          {% include "toolkit/notification-banner.html" %}
+          {% endwith %}
+      {% endfor %}
+  {% endif %}
+  {% endwith %}
+
+  {% for error in edit_admin_user_form.edit_admin_name.errors %}
+    {%
+      with
+        message = error,
+        type = "destructive"
+    %}
+      {% include "toolkit/notification-banner.html" %}
+    {% endwith %}
+  {% endfor %}
+
+  {% with heading = admin_user.emailAddress %}
+    {% include "toolkit/page-heading.html" %}
+  {% endwith %}
+
+  <form method="post">
+    <div class="grid-row">
+      <div class="column-two-thirds">
+        <input id="csrf_token" name="csrf_token" type="hidden" value="{{ csrf_token() }}">
+        {%
+          with
+          question = "Name",
+          name = edit_admin_user_form.edit_admin_name.name,
+          value = edit_admin_user_form.edit_admin_name.data,
+          error = ""
```

Comment:
Just to check my understanding, is this error blank because we're only displaying the error in the header, not next to the form field?

Reply:
I think so, but mostly I copied this question structure from add_buyer_email_domain.html template.